explorer/src/__tests__/entity/TaskRun.test.ts

```diff
@@ -33,7 +33,14 @@ describe('entity/taskRun', () => {
       [jr.id],
     )
 
-    const retrieved = await db.manager.findOne(JobRun, jr.id)
+    const retrieved = await db
```

Comment:
There's a query in the `jobRuns#show` controller that looks similar to this. Could that be extracted into a function within `entity/JobRun` and reused here?

Reply:
yeah, didn't realise that there was code duplication with prod code. will do